### PR TITLE
fix(build): fix gulp plumber

### PR DESCRIPTION
**Pull request by ncoden**
Fix gulp-plumber which prevent the container to crash, but stop the file watching

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -36,6 +36,12 @@ var config = {
   vendorDestFilename: 'vendor.js'
 };
 
+// Prevent watch end
+var onError = function (err) {
+  console.log(err);
```
Comment (iamdey) [inline gulpfile.js]: `console.error` (bien que ça ne change pas grand chose)

File: gulpfile.js [hunk 2]
```diff
@@ -36,6 +36,12 @@ var config = {
...
+  this.emit('end');
```
Comment (iamdey) [inline gulpfile.js]: c'est specific au stream ou à plumber ?

Comment (ncoden) [inline gulpfile.js]: I don't understand. It's what make this event considered as an Error, and stop the build process, without stop the file watching

File: gulpfile.js (hunk 1 above)
Comment (ncoden) [inline gulpfile.js]: done

File: gulpfile.js (hunk 2 above)
Comment (iamdey) [inline gulpfile.js]: ok sounds like it's the gulp Stream

**Pull request merged**